app/views/aid_applications/_navigation.erb

```diff
@@ -6,7 +6,7 @@
 </p>
 
 <div class="tab-bar">
-  <%= link_to organization_aid_application_edit_path(current_organization, current_aid_application), class: "tab-bar__tab #{ current_path?(organization_aid_application_edit_path(current_organization, current_aid_application)) ? 'is-selected' : '' }" do %>
+  <%= link_to edit_organization_aid_application_eligibility_check_path(current_organization, current_aid_application), class: "tab-bar__tab #{ current_path?(edit_organization_aid_application_eligibility_check_path(current_organization, current_aid_application)) || current_path?(edit_organization_aid_application_applicant_information_path(current_organization, current_aid_application)) ? 'is-selected' : '' }" do %>
```

Comment:
What would you think of having two different tabs: one for eligibility and a second for applicant info?  I suggest it for visibility that there are 2 pages, as well as making it easier to edit just the Applicant Info.

Reply:
yeah that's a great idea - i'll add an additional tab!